### PR TITLE
Add GAOREP.

### PR DESCRIPTION
**Pull request by Phrohdoh**

### CODE REVIEW

**Comment by Phrohdoh** [comment]
Updated.

**Comment by Phrohdoh** [comment]
Updated again.

File: rules/structures.yaml [hunk 1]
```diff
@@ -344,6 +344,33 @@ GATECH:
...
 
+GAOREP:
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 80
+		Prerequisites: garefn, gatech, ~structures.allies
+		BuildLimit: 1
+	Selectable:
+		Bounds: 96, 98, -2, -24
```
Comment (abcdefg30) [inline rules/structures.yaml]: Please change this to `Bounds: 134, 102, -10, -21`.

**Comment by Phrohdoh** [comment]
Updated bounds.

**Comment by abcdefg30** [comment]
:+1:

**Pull request merged**